python/lsst/ts/xml/data/sal_interfaces/Reflector/Reflector_Events.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
+<SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
+  <Enumeration>
+    ReflectorState_Off=0,
+    ReflectorState_On=1,
+    ReflectorState_Unknown=2,
+    ReflectorState_Error=3,
+  </Enumeration>
+  <SALEvent>
+    <Subsystem>Reflector</Subsystem>
+    <EFDB_Topic>Reflector_logevent_reflectorState</EFDB_Topic>
+    <Description>Reflector state change</Description>
+    <item>
+      <EFDB_Name>reflectorState</EFDB_Name>
+      <Description>Reflector state; a ReflectorState enum.</Description>
+      <IDL_Type>int</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALEvent>
```

Comment:
I think this event should be replaced with two events that indicate that the reflector is on or off. The error state that is here should be superceded by the fault state since there isn't a second discrete device that also indicates state.